db/schema.rb

```diff
@@ -82,6 +82,7 @@
   add_index "profiles", ["user_id"], name: "index_profiles_on_user_id", using: :btree
 
   create_table "users", force: :cascade do |t|
+    t.string   "email_plain",                   limit: 255, default: "", null: false
```

Comment:
it will be problematic to create *new* accounts while this schema is in force, so we should do a subsequent drop of this column as soon as possible after verifying that the migration was successful (via manual testing of login, etc).